### PR TITLE
Prep schema for 2019

### PR DESCRIPTION
**Pull request by KateUnger**
I think I found all the year-specific parts of the viewer and deleted them. Additionally, I added a couple comments concerning format or certain variables.

Let me know if this should be PR-ed into another repository.

### CODE REVIEW

File: androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/TeamTemplate.java [hunk 1]
```diff
@@ -11,51 +11,9 @@
 /**
  * Created by Teo on 1/10/2018.
  */
-@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonIgnoreProperties(ignoreUnknown = false)
```
[the remark this answers is not in the document]
Comment (ccsaposs) [reply androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/TeamTemplate.java]: Got it.  Is that something that could be separated into a different PR?

[the remark this answers is not in the document]
Comment (KateUnger) [reply androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/TeamTemplate.java]: No, they'll be changed once in a while just to check schema. For the moment it should be off.

[the remark this answers is not in the document]
Comment (ccsaposs) [reply androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/TeamTemplate.java]: Ah okay.  Will they only be changed for testing? If so, `master` should always have them off, right?

Also, should they be "off" regardless of if there's 2018 schema?

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,35 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ccsaposs) [inline .idea/misc.xml]: This is deleting the `.idea/misc.xml` from the `master` branch.  Is that your intention?

**Comment by KateUnger** [comment]
No, just from this PR

File: .idea/misc.xml (hunk 2 above)
Comment (ccsaposs) [reply .idea/misc.xml]: Moving this conversation here:
> No, just from this PR ([link](https://github.com/frc1678/viewer-2018-android/pull/121#issuecomment-450531204))

Okay, currently this PR is requesting to delete the `.idea/misc.xml` file from `master`.  @KateUnger Could you update the PR by resetting the `.idea/misc.xml` to match master?

File: androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/CalculatedTeamInMatchData.java [hunk 3]
```diff
@@ -48,5 +46,13 @@
     public Integer totalNumScaleFoul;
     public Integer numSpilledCubesAuto;
     public Integer numSpilledCubesTele;
+    public Integer rankAgility;
+    public Integer rankDefense;
+    public Integer rankSpeed;
+    public Object startingPosition;
+    public String scoutName;
```
Comment (thondascully) [inline androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/CalculatedTeamInMatchData.java]: Why are these added when they should be removed? Everything game specific (etc "cubes", "scale") should be removed for the following year's template

File: androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/Match.java [hunk 4]
```diff
@@ -37,10 +34,9 @@
     public Map<String, String> redSwitch;
     public Map<String, String> blueSwitch;
     public Map<String, String> scale;
```
Comment (thondascully) [inline androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/Match.java]: How come these were not removed as well? Everything game specific should've been removed if I recall correctly

File: androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/TeamTemplate.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (ccsaposs) [reply androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/TeamTemplate.java]: Ok cool, so it's only for testing and should be `false` on `master`.  Could you separate that into a different PR since that's not directly related with removing 2018 schema?

File: .idea/misc.xml [hunk 5]
```diff
@@ -29,7 +29,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
[the remark this answers is not in the document]
Comment (ccsaposs) [reply .idea/misc.xml]: No worries, thanks!

File: androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/TeamTemplate.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (KateUnger) [reply androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/TeamTemplate.java]: Shouldn't I leave it, since it has to do with schema, and should be off when schema is first added?

[the remark this answers is not in the document]
Comment (ccsaposs) [reply androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/TeamTemplate.java]: It isn't related to removing old schema, and should be separated into a different PR.
